erroranalysis/erroranalysis/_internal/constants.py

```diff
@@ -11,8 +11,20 @@
 SPLIT_FEATURE = 'split_feature'
 LEAF_INDEX = 'leaf_index'
 METHOD = 'method'
-METHOD_EXCLUDES = 'excludes'
-METHOD_INCLUDES = 'includes'
+
+
+class CohortFilterMethods:
+    """Cohort filter methods.
+    """
+
+    METHOD_INCLUDES = 'includes'
+    METHOD_EXCLUDES = 'excludes'
+    METHOD_EQUAL = 'equal'
+    METHOD_GREATER = 'greater'
+    METHOD_LESS = 'less'
+    METHOD_LESS_AND_EQUAL = 'less and equal'
+    METHOD_GREATER_AND_EQUAL = 'greater and equal'
```

Comment:
This should technically be less than or equal, or at least less or equal (stressing on the OR rather than AND). I know that you're just moving this around from another file, but I'm just wondering whether we're adjusting those at some point.

Reply:
this would need to be fixed everywhere, ui and backend, and wouldn't be backwards compatible, so I would be careful when making this change.  Chris also had a proposal around changing these to be more like "lt", "gt", etc, but it was not implemented, probably because it would be such a large change.

Reply:
I remember @xuke444 mentioning this last week. However, "lt" etc. aren't really readable. Not sure if that's a concern but it's worth thinking about. In any case, all of this is out of scope for this PR. Would just be nice to know what the long-term plan is on this.